### DISCUSSION TITLE
Nativescript 6 document on Nativescript website not enough any more

### DISCUSSION THREAD

**Discussion by bhcuong2008** [NativeScript/NativeScript#10458] [General]
Hi,

I have an existing NS 6 app. I've found that the document for NS 6 not enough any more. For example, RadDataform not available, https://v6.docs.nativescript.org/ui/components/DataForm/dataform-overview

I could not develop the app without API document. Please help.
Thank you.

**Comment by NathanWalker** [comment]
Hi @bhcuong2008 the Discord community, https://nativescript.org/discord, can likely help give pointers on that as well as project update pointers.

**Comment by bhcuong2008** [comment]
@NathanWalker Discord is new to me. I will take a look at it. Thank you. 
Before, it had enough document on Nativescript website, and google it with full info of NS 6, 7, 8. But now there's no result from google search.
